src/test/kotlin/com/atlassian/performance/tools/hardware/HardwareApdexAnalysisIT.kt

```diff
@@ -0,0 +1,269 @@
+package com.atlassian.performance.tools.hardware
+
+import com.amazonaws.services.ec2.model.InstanceType
+import com.amazonaws.services.ec2.model.InstanceType.*
+import com.amazonaws.services.s3.transfer.TransferManagerBuilder
+import com.atlassian.performance.tools.hardware.IntegrationTestRuntime.aws
+import com.atlassian.performance.tools.hardware.IntegrationTestRuntime.logContext
+import com.atlassian.performance.tools.hardware.IntegrationTestRuntime.taskName
+import com.atlassian.performance.tools.hardware.IntegrationTestRuntime.workspace
+import com.atlassian.performance.tools.hardware.failure.BugAwareTolerance
+import com.atlassian.performance.tools.jiraactions.api.*
+import com.atlassian.performance.tools.jirasoftwareactions.api.actions.ViewBacklogAction
+import com.atlassian.performance.tools.jvmtasks.api.TaskTimer.time
+import com.atlassian.performance.tools.lib.Apdex
+import com.atlassian.performance.tools.lib.table.GenericPlainTextTable
+import com.atlassian.performance.tools.lib.WeightedActionMetric
+import com.atlassian.performance.tools.lib.readResult
+import com.atlassian.performance.tools.lib.s3cache.S3Cache
+import com.atlassian.performance.tools.report.api.StandardTimeline
+import com.atlassian.performance.tools.report.api.result.EdibleResult
+import com.atlassian.performance.tools.report.api.result.RawCohortResult
+import com.atlassian.performance.tools.workspace.api.TestWorkspace
+import org.apache.logging.log4j.Logger
+import org.junit.Test
+import java.io.File
+import java.time.Duration
+import java.nio.file.FileSystems
+import java.nio.file.Paths
+
+
+class HardwareApdexAnalysisIT {
+
+    private val logger: Logger = logContext.getLogger(this::class.java.canonicalName)
+
+    // define the appdex sets we want to investigate
+    private val jiraInstanceTypeOfInterest: InstanceType = C518xlarge
+    private val nodeCountOfInterest: Int = 7
+
+    // The following needs to match the configuration of the test run you want to analyze
+    // Taken from extraLarge.Kt hold + ramp + flat values
+    private val testDuration: Duration = Duration.ZERO + Duration.ofSeconds(90) + Duration.ofMinutes(20)
```

Comment:
Yes I did try that and then consciously went back to this, since using extraLarge() requires a Jira Licence, which seemed overkill for this kind of test/process

```
java.lang.Exception: Put a Jira license to /Volumes/PROJECTS/projects/github.com/mminns/jira-hardware-exploration/jira-license.txt

	at com.atlassian.performance.tools.hardware.HwrDatasetCatalogue.overrideLicense(HwrDatasetCatalogue.kt:162)
```

Reply:
Ooooh, I see. Ok